# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.h
# Commit: 67f28439cbc7
# Full Hash: 67f28439cbc71117a690ea3eee3ef12b0593a7e6
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-29 20:54 +0000
# Regressor Bug: 1987945
# File Overlap Count: 1
# Overlapping Files: accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P3: Support details relation in anchored targets in remote accessibles. r=Jamie,layout-anchor-positioning-reviewers,layout-reviewers,emilio,firefox-style-system-reviewers
# ==============================================================================

diff -r b93b02b24271 -r 67f28439cbc7 layout/base/PresShell.h
--- a/layout/base/PresShell.h	Mon Sep 29 20:54:29 2025 +0000
+++ b/layout/base/PresShell.h	Mon Sep 29 20:54:29 2025 +0000
@@ -774,6 +774,11 @@
   }
 
   inline void RemoveAnchorPosPositioned(nsIFrame* aFrame) {
+#ifdef ACCESSIBILITY
+    if (nsAccessibilityService* accService = GetAccService()) {
+      accService->NotifyAnchorPositionedRemoved(this, aFrame);
+    }
+#endif
     mAnchorPosPositioned.RemoveElement(aFrame);
   }
 